ern-runner-gen-android/src/AndroidRunnerGenerator.ts

```diff
@@ -19,12 +20,33 @@ export default class AndroidRunnerGenerator implements RunnerGenerator {
   }
 
   public async generate(config: RunnerGeneratorConfig): Promise<void> {
+    log.info(`generate config ${JSON.stringify(config)}`)
```

Comment:
Could you please move this one to `debug` log level rather than `info`

Reply:
I think, I will remove this. Not really needed.